src/ecredis_command_parser.erl

```diff
@@ -135,6 +141,13 @@ check_sanity_of_keys([Term1, _Term2 | Rest]) ->
             get_keys_from_rest(Rest);
         "evalsha" ->
             get_keys_from_rest(Rest);
+        "memory" ->
+            %% duplicate logic similar to get_key_from_command.
+            case Rest of
+                [Term3 | _] when is_binary(Term3) -> binary_to_list(Term3);
+                [Term3 | _] when is_list(Term3) -> Term3;
+                _ -> undefined
+            end;
         _ ->
             [undefined]
```

Comment:
should this be undefined?

Reply:
yeah - basically we dont handle those unknown commands yet.

Reply:
what I meant was: [undefined] -> undefined

Reply:
i think everything else should also be a list here. fixed it.